src/libs/ReportUtils.ts

```diff
@@ -1941,22 +1749,26 @@ function getReportName(report, policy = undefined) {
     }
 
     // Not a room or PolicyExpenseChat, generate title from participants
-    const participantAccountIDs = (report && report.participantAccountIDs) || [];
-    const participantsWithoutCurrentUser = _.without(participantAccountIDs, currentUserAccountID);
+    const participantAccountIDs = report?.participantAccountIDs ?? [];
+    const participantsWithoutCurrentUser = participantAccountIDs.filter((accountID) => accountID !== currentUserAccountID);
     const isMultipleParticipantReport = participantsWithoutCurrentUser.length > 1;
 
-    return _.map(participantsWithoutCurrentUser, (accountID) => getDisplayNameForParticipant(accountID, isMultipleParticipantReport)).join(', ');
+    return participantsWithoutCurrentUser.map((accountID) => getDisplayNameForParticipant(accountID, isMultipleParticipantReport)).join(', ');
 }
 
 /**
  * Recursively navigates through thread parents to get the root report and workspace name.
  * The recursion stops when we find a non thread or money request report, whichever comes first.
- * @param {Object} report
- * @returns {Object}
  */
-function getRootReportAndWorkspaceName(report) {
+function getRootReportAndWorkspaceName(report?: OnyxEntry<Report>) {
```

Comment:
we need to leave this type `undefined` because for example 
<img width="755" alt="image" src="https://github.com/Expensify/App/assets/25230417/bcf93b99-88ba-489c-a520-497f1929c53a">